src/components/editor/EditAuthorsForm.jsx

```diff
@@ -84,8 +41,8 @@ export default class EditAuthorsForm extends BaseComponent {
               <div key={author.slug}>
                 {
                   !striked
-                    ? <a style={{float: "left", cursor: "pointer"}} aria-label="Remove Author Button" onClick={this.handleDeleteAuthor.bind(this, author.slug, true)}>&times;&nbsp;</a>
-                    : <a style={{float: "left", cursor: "pointer"}} aria-label="Remove Author Button" onClick={this.handleUnstrikeAuthor.bind(this, author.slug)}>~&nbsp;</a>
+                    ? <a style={{float: "left", cursor: "pointer"}} aria-label="Remove Author Button" onClick={this.props.handleDeleteAuthor.bind(null, author.slug, true)}>&times;&nbsp;</a>
+                    : <a style={{float: "left", cursor: "pointer"}} aria-label="Remove Author Button" onClick={this.props.handleUnstrikeAuthor.bind(null, author.slug)}>~&nbsp;</a>
```

Comment:
Okay, I'll figure something out, makes sense. Since I already put together two of them, I can just as well add the last one in.